.gitlab-ci.yml

```diff
@@ -49,23 +38,53 @@ build-msi:
       - wine-build/*.exe
       - wine-build/*.msi
 
+
+pull:
+  rules:
+    - if: '$CI_PIPELINE_SOURCE == "trigger" && $CI_COMMIT_BRANCH == "master"'
+  image: nitrokey/ci-pull:latest
+  tags:
+    - docker
+  stage: pull-github
+  variables:
+    #Repo for shared scripts (pull.sh release.sh, nightly_upload.sh):
+    SCRIPTS_REPO: git@git.dotplex.com:nitrokey/gitlab-ci.git
```

Comment:
Can this be provided by a variable? If not, can a separate script export these instead? 
Right now indeed its duplicated, while should be modifiable from a single place.

Reply:
i can use one global variable section:
https://github.com/Nitrokey/pynitrokey/commit/1eb146180056aa6b943d90265cbf5308937b89ad

using another variable will lead to more manual  work in the Webinterface

Reply:
Looks better!